src/components/article/LeadingImage.js

```diff
@@ -34,12 +50,17 @@ class LeadingImage extends React.Component {
         break
     }
 
+    const defaultImgUrl = replaceStorageUrlPrefix(_.get(image, 'tiny.url'))
+
+    const imgJsx = this.state.isMount ? (
+      <img src={defaultImgUrl} alt={description} srcSet={getImageSrcSet(image)} />
+    ) : (
+      <img src={defaultImgUrl} alt={description} />
```

Comment:
Yes. I mean maybe you could add a blurry image class to the first loaded tiny image.